connectors/citrus-openapi/src/main/resources/META-INF/citrus/message/validator/openApi

```diff
@@ -0,0 +1,2 @@
+name=defaultOpenApiMessageValidator
+type=org.citrusframework.openapi.validation.OpenApiSchemaValidation
```

Comment:
During review it felt a bit weird that we have the same class declared as a schema validator and message validator. First I thought this is a copy and paste error, but I believe this is meant to be on purpose, right?